pydbus/strong_typing.py

```diff
@@ -34,3 +35,10 @@ def decorate(func):
 		verify_arguments(func)
 		return func
 	return decorate
+
+
+class TypedSignal(signal):
+
+	def __init__(self, arguments):
```

Comment:
I'm not in love with typed_method's interface, but as it's only a fallback for people that can't use Python 3, I don't really care. On the other hand, typed_signal should be pretty.